plugins/woocommerce/changelog/add-delayed-account-creation-block-50632

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I'd argue this could be a major? It's a chunky bit of new functionality

Reply:
"added or deprecated functionality in a backwards compatible way" is minor.